packages/common/src/system/guid.spec.ts

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import 'reflect-metadata';
+
+import { Guid } from './guid';
+
+describe('createGuid()', () => {
+    it('create new UUID', () => {
+        const guid = Guid.createGuid();
+        expect(guid.length).toEqual(36);
+        expect(guid.substr(13, 2)).toEqual('-6');
+    });
+
+    it('create new UUID with fixed node part', () => {
+        const guid = Guid.createGuid();
+        const nextGuid = Guid.createGuid(guid);
+        expect(guid.length).toEqual(36);
+        expect(guid.substr(24, 6)).toEqual(nextGuid.substr(24, 6));
```

Comment:
>  expect(guid.substr(24, 6)).toEqual(nextGuid.substr(24, 6)); [](start = 6, length = 61)

also validate nextGuid is not same as guid

Reply:
updated